editor.js

```diff
@@ -82,7 +82,7 @@ function makeEditorShim(id) {
 
 
 function extension_editor_finish_run() {
-	if (typeof(unsafeWindow) !== 'undefined') {
+	if ((typeof(unsafeWindow) !== 'undefined') && (navigator.userAgent.indexOf('Mobile') === -1)) {
```

Comment:
XKit.browser().mobile is in one of the internal extensions, I think? It also relys on Tumblr detecting the browser (so things don't break if you request the desktop site), so I don't think it would work here.